src/chapters/booleans-and-conditionals/logical-operators.rst

```diff
@@ -0,0 +1,273 @@
+
+.. sourcecode:: js
+
+   ((x * 5) >= 10) && ((y - 6) <= 20)
+
+While parentheses make expression more readable, they are not strictly necessary due to default operator precedence.
+
+The following table lists operators in order of precedence, from highest (applied first) to lowest (applied last). A complete table for the entire language can be found in the `MDN JavaScript Documentation <https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_Precedence#Table>`_.
+
+.. list-table:: Operator Precedence
+   :widths: auto
+   :header-rows: 1
+
+   * - Precedence
+     - Category
+     - Operators
+   * - (highest)
+     - Logical NOT
+     - ``!``
+   * -
+     - Exponentiation
+     - ``**``
+   * -
+     - Multiplication and division
+     - ``*``, ``/``, ``%``
+   * -
+     - Addition and subtraction
+     - ``+``, ``-``
+   * -
+     - Comparison
+     - ``<=``, ``>=``, ``>``, ``<``
+   * -
+     - Equality
+     - ``===``, ``!==``, ``==``, ``!=``
+   * -
+     - Logical AND
+     - ``&&``
+   * - (lowest)
+     - Logical OR
+     - ``||``
+
+Truth Tables
+------------
+
+.. index:: ! truth table
+
+**Truth tables** help us understand how logical operators work by calculating all of the possible return values of a boolean expression. Let's look at the truth table for ``&&``, which displays assumes we have two boolean expressions, A and B, joined by ``&&``.
+
+.. admonition:: Question
+
+   .. list-table:: Truth Table for ``&&``
+      :widths: auto
+      :header-rows: 1
+
+      * - A
+        - B
+        - A ``&&`` B
+      * - ``true``
+        - ``true``
+        - ``true``
+      * - ``true``
+        - ``false``
+        - ``false``
+      * - ``false``
+        - ``true``
+        - ``false``
+      * - ``false``
+        - ``false``
+        - ``false``
+
+Consider the first row of the truth table. This row states that if A is true and B is true, then A && B is true. This is a fact, regardless of what boolean expressions A and B might actually be. The two middle rows demonstrate that if either A or B is false, then A && B is false. (If this idea is hard to grasp, try substituting actual expressions for A and B.)
+
+Check Your Understanding
+------------------------
+
+.. admonition:: Question
+
+   Complete the truth table for ``||``.
+
+   .. list-table:: Truth Table for ``||``
+      :widths: auto
+      :header-rows: 1
+
+      * - A
+        - B
+        - A ``||`` B
+      * - ``true``
+        - ``true``
+        -
+      * - ``true``
+        - ``false``
+        -
+      * - ``false``
+        - ``true``
+        -
+      * - ``false``
+        - ``false``
+        -
+
+.. admonition:: Question
+
+   Which of the following properly expresses the precedence of operators (using parentheses) in the following expression?
```

Comment:
Rephrasing: Replace "precedence of operators" with "order of operations", which is a more common term.